content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,130 +2,181 @@
 Getting started with Helpdesk
 =============================
 
-Odoo :guilabel:`Helpdesk` is a ticket-based customer support application. Teams can track,
-prioritize, and solve customer issues from their pipeline, which is organized in customizable
-stages. Multiple teams can be configured and managed in one dashboard.
+Odoo *Helpdesk* is a ticketing-based customer support application. Multiple teams can be configured
+and managed in one dashboard, each with their own pipeline for tickets submitted by customers.
+Pipelines are organized in customizable stages that enable teams to track, prioritize, and solve
+customer issues quickly and efficiently.
 
-Create Helpdesk teams
-=====================
+Create a Helpdesk team
+======================
 
-Setting up multiple teams allows for tickets to be grouped by location or by support type.
-
-To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`. To
-create a new team, click on the :guilabel:`New` button in the top left of the dashboard. From there,
-name the new team, and fill out the remaining fields as defined in the following sections on the
-form.
+To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams`. To create a new team, click the :guilabel:`New` button in the top-left of the
+dashboard.
 
 .. image:: getting_started/helpdesk-teams-list.png
    :align: center
-   :alt: View of the Helpdesk teams page in Odoo Helpdesk
+   :alt: View of the Helpdesk teams page in Odoo Helpdesk.
+
+On the blank Helpdesk team form, enter a :guilabel:`Name` for the new team. Then, enter a
+description of the team in the field below the team name, if desired. To change the company this
+team is assigned to, select it from the :guilabel:`Company` drop-down.
```

Comment:
```suggestion
team is assigned to, select it from the :guilabel:`Company` drop-down menu.
```

Reply:
If memory serves, I believe we're always supposed to specify the "drop-down" element being utilized, be it a window, form, menu, etc.